Hardened profiles now set -D_FORTIFY_SOURCE=3, do not let upstream weaken it.

--- a/configure.ac
+++ b/configure.ac
@@ -206,18 +206,6 @@
 fi
 
 dnl -----------------------------------------------
-dnl Check for GCC -D_FORTIFY_SOURCE support
-dnl -----------------------------------------------
-AC_MSG_CHECKING(for gcc support of FORTIFY_SOURCE)
-TMPCFLAGS="${CFLAGS}"
-CFLAGS="${CFLAGS} -D_FORTIFY_SOURCE=2"
-AC_TRY_COMPILE(,,[gcc_have_fortify_source=yes],[gcc_have_fortify_source=no])
-AC_MSG_RESULT($gcc_have_fortify_source)
-if test "$gcc_have_fortify_source" != "yes"; then
-    CFLAGS="${TMPCFLAGS}"
-fi
-
-dnl -----------------------------------------------
 dnl Check for GCC -Wformat-security support
 dnl -----------------------------------------------
 AC_MSG_CHECKING(for gcc support of -Wformat -Wformat-security)
